recipes/MOABB/MotorImagery_decoding/BNCI2014001/train.yaml

```diff
@@ -0,0 +1,144 @@
+###################################################
+#
+# Author
+# ------
+# Davide Borra, 2021
+###################################################
+seed: 2602
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+
+# DIRECTORIES
+data_folder: !PLACEHOLDER  #'/path/to/MOABB_BNCI2014001'
+output_folder: '/home/prometheus/Documents/results/MOABB_BNCI2014001' #!ref results/MOABB_BNCI2014001/<seed>
```

Comment:
I think this should be `output_folder: results/MOABB_BNCI2014001/<seed>`. Otherwise, it is hardcoded.